### PR TITLE
Added error message if no views are active

### PR DESCRIPTION
**Pull request by justin0022**
Addresses #574

### CODE REVIEW

File: assets/src/containers/IndexPage.js [hunk 1]
```diff
@@ -24,6 +26,9 @@ const styles = theme => ({
 function IndexPage (props) {
   const { classes, courseInfo, courseId } = props
 
+  if (isObjectEmpty(courseInfo.course_view_options))
+    return (<Error>No data visualizations have been added for this course.</Error>)
```
Comment (pushyamig) [inline assets/src/containers/IndexPage.js]: the `course_view_options` is not empty when all the views are disabled. This is how the part of Json looks like `"course_view_options": {"fa": 0, "ap": 0, "gd": 0}}`. This info is pulled from the DB.  The code never enter this case
There is error case(when DB look up failed) the Json might looks like this`"course_view_options": ""` 
I think it needs to support both the cases.

Comment (justin0022) [reply assets/src/containers/IndexPage.js]: Ah good point, okay, I'll fix this.

**Comment by justin0022** [comment]
@pushyamig I now check to see if every value of the `course_view_options` are 0 before displaying the message.

**Comment by pushyamig** [comment]
>@pushyamig I now check to see if every value of the course_view_options are 0 before displaying the message.

Really Nice you have added the test. This looks good!
I feel how to run the test from the command line was never added to README. It will be useful to have the info there so that encourages people to add more tests.

I feel the Backend should also start adding testing as Frontend does :(

**Pull request merged**